client/src/components/ExportCsv.tsx

```diff
@@ -1,46 +1,90 @@
-import { ExportToCsv } from "export-to-csv"
+import {formatIsoDate} from 'date/utils';
+import {ExportToCsv} from 'export-to-csv';
 
-function csvExporter(user: any) {
-    const data = [
-        {
-            displayName: user.displayName,
-            approvals: user.aggregatedAuthoredTotals.approvals,
-            rejections: user.aggregatedAuthoredTotals.rejections,
-            totalComments: user.aggregatedAuthoredTotals.commentsWrittenTotal,
-            commentsToOthers: user.aggregatedAuthoredTotals.commentsWrittenToOthers,
-            changeRequests: user.aggregatedAuthoredTotals.changesCreated,
-            commits: user.aggregatedAuthoredTotals.commits,
-            averageTimeInReview: user.aggregatedAuthoredTotals.meanChangeOpenTimeMsec,
-            commentsPerRequest: user.aggregatedAuthoredTotals.commentsWrittenToOthers / user.aggregatedReceivedTotals.reviewRequests,
-          },
-        // "test",
-    ];
-    const options = { 
-        fieldSeparator: ',',
-        quoteStrings: '"',
-        decimalSeparator: '.',
-        showLabels: true, 
-        showTitle: true,
-        title: 'User Code Review Stats',
-        filename: `${user.displayName}-stats.csv`,
-        useTextFile: false,
-        useBom: true,
-        useKeysAsHeaders: true,
-      };
-    const csvExporter = new ExportToCsv(options);
-    csvExporter.generateCsv(data);
-}
-
-const ExportCsv = ( {user} : any ) => {
+const sum = (acc: number, item: number) => acc + item;
```

Comment:
This is copypasted from another file. Instead of doing that, we should add this function in a utility file, export it, and use it both here and in AllUsersOverviewPage.

Reply:
You're right, I have created a `utils.ts` file in utils directory